packages/salesforcedx-utils-vscode/src/telemetry/telemetryReporter.ts

```diff
@@ -10,7 +10,7 @@ import * as os from 'os';
 import * as path from 'path';
 import { Disposable, env, UIKind, version, workspace } from 'vscode';
 
-export default class TelemetryReporter extends Disposable {
+export class TelemetryReporter extends Disposable {
```

Comment:
Small change but this allows the class to be discoverable by the TypeScript lang server when trying to import it somewhere else.